dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1140,113 +1151,127 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
             Logger.info(this, "Found :\t" + l.size() + " " + _className + "(s)");
             if(address){
                 for (int j = 0; j < l.size(); j++) {
-                    HashMap<String, String> dcResults = (HashMap<String,String>)l.get(j);
-                    DotConnect dc = new DotConnect();
-                    dc.setSQL("insert into address values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)");
-                    dc.addParam(dcResults.get("addressid"));
-                    dc.addParam(dcResults.get("companyid"));
-                    dc.addParam(dcResults.get("userid"));
-                    dc.addParam(dcResults.get("username"));
-                    dc.addParam(dcResults.get("createDate"));
-                    dc.addParam(dcResults.get("ModifiedDate"));
-                    dc.addParam(dcResults.get("classname"));
-                    dc.addParam(dcResults.get("classpk"));
-                    dc.addParam(dcResults.get("description"));
-                    dc.addParam(dcResults.get("street1"));
-                    dc.addParam(dcResults.get("street2"));
-                    dc.addParam(dcResults.get("city"));
-                    dc.addParam(dcResults.get("state"));
-                    dc.addParam(dcResults.get("zip"));
-                    dc.addParam(dcResults.get("country"));
-                    dc.addParam(dcResults.get("phone"));
-                    dc.addParam(dcResults.get("fax"));
-                    dc.addParam(dcResults.get("cell"));
-                    dc.addParam(UtilMethods.isSet(dcResults.get("priority")) ? Integer.parseInt(dcResults.get("priority")) : null);
-                    dc.getResults();
+                    final HashMap<String, String> dcResults = (HashMap<String,String>)l.get(j);
+                    LocalTransaction.wrap(() -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this lambda expression number of lines from 24 to at most 20. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1188)